homeassistant/components/playstation_network/entity.py

```diff
@@ -0,0 +1,33 @@
+"""Base entity for PlayStation Network Integration."""
+
+from typing import TYPE_CHECKING
+
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.entity import EntityDescription
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import PlaystationNetworkCoordinator
+
+
+class PlaystationNetworkServiceEntity(CoordinatorEntity[PlaystationNetworkCoordinator]):
+    """Common entity class for PlayStationNetwork Service entities."""
+
+    coordinator: PlaystationNetworkCoordinator
+    _attr_has_entity_name = True
+
+    def __init__(self, coordinator, entity_description: EntityDescription) -> None:
```

Comment:
the previous suggestion was wrong 🙈
```suggestion
    _attr_has_entity_name = True

    def __init__(
        self,
        coordinator: PlaystationNetworkCoordinator,
        entity_description: EntityDescription,
    ) -> None:
```